lib/NeuraDialect/Transforms/MapToAcceleratorPass.cpp

```diff
@@ -47,46 +71,52 @@ struct MapToAcceleratorPass
       RecurrenceCycle *longest = nullptr;
       int rec_mii = 1;
       for (auto &cycle : recurrence_cycles) {
+        llvm::outs() << "[DEBUG] Recurrence cycle (length " << cycle.length
+                     << "):\n";
+        for (Operation *op : cycle.operations) {
+          llvm::outs() << "  " << *op << "\n";
+        }
         if (!longest || cycle.length > longest->length)
           longest = &cycle;
       }
 
       if (longest) {
-        llvm::errs() << "[MapToAcceleratorPass] Longest recurrence cycle (length "
-                    << longest->length << "):\n";
+        llvm::outs()
+            << "[MapToAcceleratorPass] Longest recurrence cycle (length "
+            << longest->length << "):\n";
         for (Operation *op : longest->operations) {
-          op->print(llvm::errs()), llvm::errs() << "\n";
+          op->print(llvm::outs()), llvm::outs() << "\n";
```

Comment:
This would dump into the `.mlir`, which is the difference from the previous `llvm::errs()`?